jsLearn/logicalOperators.js

```diff
@@ -0,0 +1,28 @@
+// 1.
+
+let age = prompt("write age", 20);
+let result = age >= 14 && age <= 90;
+
+
+// 2
+
+let age = prompt("write age", 20);
```

Comment:
double declaration of age

Reply:
age is string but used as a number, convert the returned value of prompt into number for safety